cuda/base/cusparse_bindings.hpp

```diff
@@ -886,7 +459,7 @@ inline void destroy(cusparseSpMatDescr_t descr)
 }
 
 
-#if (CUDA_VERSION >= 11031)
+#if defined(CUDA_VERSION) && (CUDA_VERSION >= 11031)
```

Comment:
Not sure if we even need these - the file should fail to compile if it's not compiled by nvcc or not including the necessary headers.

Reply:
I would just leave it, it's not hurting.